dist/vl-grid.js

```diff
@@ -232,6 +252,24 @@ export class VlColumn extends NativeVlElement(HTMLDivElement) {
         VlColumn.__sizeClass(newValue, this._maxSizeAttribute)
     );
   }
+
+  _configureDefaults() {
+    if (!this.hasAttribute('size')) {
+      this._sizeChangedCallback(null, this._sizeAttribute);
+    }
+
+    if (!this.hasAttribute('medium-size')) {
+      this._medium_sizeChangedCallback(null, this._mediumSizeAttribute);
+    }
+
+    if (!this.hasAttribute('small-size')) {
+      this._small_sizeChangedCallback(null, this._smallSizeAttribute);
+    }
+
+    if (!this.hasAttribute('medium-size')) {
```

Comment:
```suggestion
    if (!this.hasAttribute('extra-small-size')) {
```
copy paste fout
eventueel ook eens kijken waarom er geen test faalt hierdoor